tslearn/metrics.py

```diff
@@ -362,10 +361,44 @@ def njit_accumulated_matrix_subsequence(subseq, longseq):
 
 
 @njit()
-def _return_path_subsequence(acc_cost_mat):
+def calculate_subsequence_path(acc_cost_mat, idx_path_end):
+    """Compute the optimal path through a accumulated cost matrix given the
+     endpoint of the sequence.
+
+    Parameters
+    ----------
+    acc_cost_mat: array, shape = (sz1, sz2)
+        The accumulated cost matrix comparing subsequence from a longer sequence
+    idx_path_end: int
+        The end position of the matched subsequence in the longer sequence.
+
+    Returns
+    -------
+    path: list of tuples of integer pairs
+        Matching path represented as a list of index pairs. In each pair, the
+        first index corresponds to `subseq` and the second one corresponds to
+        `longseq`. The startpoint of the Path is :math:`P_0 = (0, ?)` and it
+        ends at :math:`P_0 = (0, idx_path_end)`
```

Comment:
Again, I disagree (cf comment above): the last element in the path should be of the form `(sz1-1, ?)`